userspace/openpilot_dependencies.sh

```diff
@@ -56,6 +56,8 @@ apt-get install --no-install-recommends -yq \
     ocl-icd-opencl-dev \
     opencl-headers \
     pkg-config \
+    libdbus-glib-1-dev \
+    libgirepository1.0-dev \
```

Comment:
these are just build time dependencies of PyQt5? if so, these shouldn't end up in the final image. see how `agnos-compiler` is used for some other stuff

Reply:
@adeebshihadeh not for PyQt5 but for dbus-python in pyproject.toml.

Reply:
@adeebshihadeh not for PyQt5 but for dbus-python in pyproject.toml. In total about 71 build time dependencies are getting installed besides the dbus-python libraries. The build time dependencies can be removed by calling purge in DockerFile but I recommend to handle build-time optimizations in another PR.